exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,11 @@
-const name = " Daniel  ";
+const myName = " Catalina-Alexandra  ";
+const trimmed = myName.trim;
+var message1 = "My name is ";
+var message2 = " and my name is ";
+var message3 = " characters long";
+
+var nameLength = myName.length;
+var message = message1 + trimmed + message2 + nameLength + message3;
```

Comment:
I am just suggesting a shorter way, you could have written this, however all the variables may have made it clearer.  for the trimmed one u could have put myName.trim straight into the message statement. Also u could maybe have just used variables for things that may change. But what u did is also not wrong. Here is just another alternative.

var message = "My name is" + myName.trim + " and my name is " + myName.length + "characters long";